packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -0,0 +1,245 @@
+
+import { Construct, Duration, IResource, Resource, Stack } from '@aws-cdk/core';
+import { CfnJobDefinition } from './batch.generated';
+import { IJobDefinitionContainer, IJobDefinitionImageConfig, JobDefinitionImageConfig } from './job-definition-image-config';
+
+/**
+ * Construction properties of the {@link JobDefinition} construct.
+ */
+export interface JobDefinitionProps {
+  /**
+   * The name of the job definition.
+   *
+   * Up to 128 letters (uppercase and lowercase), numbers, hyphens, and underscores are allowed.
+   *
+   * @default Cloudformation-generated name
+   */
+  readonly jobDefinitionName?: string;
+
+  /**
+   * An object with various properties specific to container-based jobs.
+   *
+   * @default - undefined
```

Comment:
This is not an optional property, so this `@default` seems incorrect...

Reply:
Got it. Fixing it 👍